string initDate
string FinishDate 

---
bool result